pkg/test/framework/runtime/components/environment/kube/settings.go

```diff
@@ -76,6 +76,8 @@ var (
 		"global.mtls.enabled":         "true",
 		"galley.enabled":              "true",
 	}
+
+	helmOverridesByTest = map[string]string{}
```

Comment:
shouldn't per-test overrides be using a non-global variable?

Reply:
Ideally yes, I don't know what data structure/interface we can add this customization. Any suggestions?

And to be more precise, we found we normalize istio deployment to be a system scope component and always run once for each `TestMain`, so it's more of per test suite settings?

I was thinking about adding optional constructor parameter to `component.Start()` interface but that seems a overkill and requires lots of changes..